CHANGELOG.md

```diff
@@ -34,11 +34,14 @@ _None._
 
 ### Breaking Changes
 
-_None._
+- `prepare_windows_host_for_node.ps1` no longer install Node.js.
+  Clients should use [`nvm-buildkite-plugin`](https://github.com/Automattic/nvm-buildkite-plugin) instead [#144]
```

Comment:
Markdown renders know how to parse this syntax as a single line:

<img width="1529" alt="image" src="https://github.com/user-attachments/assets/c95dffba-d35c-4081-a9ad-985325dfc75b" />